src/Compilers/VisualBasic/Portable/Binding/MemberSemanticModel.vb

```diff
@@ -794,6 +794,51 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         Friend Overrides Function GetOperationWorker(node As VisualBasicSyntaxNode, options As GetOperationOptions, cancellationToken As CancellationToken) As IOperation
+            Dim argumentNode = TryCast(node, ArgumentSyntax)
+            If argumentNode IsNot Nothing Then
```

Comment:
Why do we only check for argument syntax here but check both syntax and parent operation in C#? If call to `GetBoundNodeSummaryAndCreateOperation` inside `TryGetOperationForArgument` is not an `IHasArgumentsExpression`, `GetOperation` will return null, whereas in C#, it will return an operation from the bound node. Is this intended?